packages/nimble-components/src/wafer-map/data-types/WaferMapColorsScale.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
The file names for these classes should use `hyphen-case` rather than `PascalCase` to match the rest of the repo.

Reply:
I updated